app/models/agents/rss_agent.rb

```diff
@@ -154,13 +159,17 @@ def check_urls(urls)
       log "Fetched #{urls.to_sentence} and created #{created_event_count} event(s)."
     end
 
+    def remembered_id_count
+      (options['remembered_id_count'].presence || 500).to_i
+    end
+
     def check_and_track(entry_id)
       memory['seen_ids'] ||= []
       if memory['seen_ids'].include?(entry_id)
         false
       else
         memory['seen_ids'].unshift entry_id
-        memory['seen_ids'].pop if memory['seen_ids'].length > 500
+		    memory['seen_ids'].pop(memory['seen_ids'].length - remembered_id_count) if memory['seen_ids'].length > remembered_id_count
```

Comment:
It looks like the indentation is still messed up here. Would you mind fixing it?